packages/typescript-config/tsconfig.json

```diff
@@ -15,6 +15,9 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "incremental": true
+    "incremental": true,
+    "paths": {
+      "@umami/jest-config": ["../../packages/jest-config/*"]
```

Comment:
Hmm, why?

Reply:
The paths option allows typescript to resolve imported internal packages